datasets/cedr/cedr.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""CEDR dataset"""
+
+import json
+import os
+
+import datasets
+
+
+# TODO: Add BibTeX citation
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@article{sboev2021data,
+  title={Data-Driven Model for Emotion Detection in Russian Texts},
+  author={Sboev, Alexander and Naumov, Aleksandr and Rybka, Roman},
+  journal={Procedia Computer Science},
+  volume={190},
+  pages={637--642},
+  year={2021},
+  publisher={Elsevier}
+}
+"""
+
+_LICENSE = """http://www.apache.org/licenses/LICENSE-2.0"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+This new dataset is designed to solve emotion recognition task for text data in Russian. The Corpus for Emotions Detecting in
+Russian-language text sentences of different social sources (CEDR) contains 9410 sentences in Russian labeled for 5 emotion
+categories. The data collected from different sources: posts of the LiveJournal social network, texts of the online news
+agency Lenta.ru, and Twitter microblog posts. There are two variants of the corpus: main and enriched. The enriched variant
+is include tokenization and lemmatization. Dataset with predefined train/test splits.
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://github.com/sag111/CEDR"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = {
+    "main": "https://drive.google.com/u/0/uc?id=1flukWj075o7InAlBZm7htXEqZM6D-LfA&export=download",
+    "enriched": "https://drive.google.com/u/0/uc?id=1w9fJf9SwxtEABy5uhougn_2ZlxKBbZHV&export=download",
```

Comment:
It's ok to use google drive but note that google drive has quotas issues that may prevent some users from downloading the dataset.

If there is another host than google drive that has the dataset we should use it, otherwise we can stick with google drive..

Reply:
Done. Changed the host for the files.